libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts

```diff
@@ -1160,6 +1162,15 @@ export class ParentalLeaveService extends BaseTemplateApiService {
     return periods
   }
 
+  checkActionName = (application: ApplicationWithAttachments) => {
+   const answer = application.answers
+   const { actionName } = answer
```

Comment:
Please add 'actionName' on function 'getApplicationAnswers' on 'parentalLeaveUtils.ts' file.
Then get it from there